pkgs/by-name/pd/pdfarranger/package.nix

```
@@ -20,7 +21,18 @@ python3Packages.buildPythonApplication rec {
     hash = "sha256-94qziqJaKW8/L/6+U1yojxdG8BmeAStn+qbfGemTrVA=";
   };
 
-  nativeBuildInputs = [ wrapGAppsHook3 ];
+  nativeBuildInputs =
+    [ wrapGAppsHook3 ]
+    ++ lib.optionals pkgs.stdenv.isDarwin [
+      pkgs.gettext
+    ];
+
+  patchPhase =
+    ''''
+    + lib.optionals pkgs.stdenv.isDarwin ''
```

```suggestion
  postPatch = lib.optionals pkgs.stdenv.isDarwin ''
```

this way adding `patches =` (for example in an override) still works.

---

actually this should use optionalString, not optionals. I wonder how this slipped through nix-review...